lib/adapters/api.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+let juttle_utils = require('../runtime/juttle-utils');
+
+//
+// The AdapterAPI is the exported interface to the juttle compiler / runtime
+// that is expected to be used by all adapters.
+//
+// The version field corresponds to the juttle version in which the latest
+// changes were made to the adapter API.
+//
+let AdapterAPI = {
+    version: '0.5.0',
+    AdapterRead: require('./adapter-read'),
+    AdapterWrite: require('./adapter-write'),
+    compiler: {
+        ASTVisitor: require('../compiler/ast-visitor'),
+        FilterJSCompiler: require('../compiler/filters/filter-js-compiler')
+    },
+    errors: require('../errors'),
+    getLogger: require('../logger').getLogger,
+    runtime: {
```

Comment:
The Elastic adapter also does `var reducers = require('juttle/lib/runtime/reducers').reducers;`, though it's just used to extract the right default value for a given reducer. Should we add that the reducers here or shred that dependency?